.github/workflows/gradle.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  pull_request:
+    branches: 
+    - develop
+    - main
+    - feature/* 
+    
+    
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: ./gradlew build
```

Comment:
Need to exclude `web:build` because it's not necessary: 
`./gradlew build -x web:build`

Reply:
I'm not understand you, should we run web build or we shouldn't? As a variant we could create step for each build action

Reply:
I think that the process of building the `web` will take a lot of time, since fetching the dependencies in `node_modules` is a very long operation. But for now we can ignore this fact